src/vs/workbench/contrib/preferences/browser/settingsWidgets.ts

```diff
@@ -329,16 +341,18 @@ export abstract class AbstractListSettingWidget<TDataItem extends object> extend
 		this.container.classList.toggle('setting-list-hide-add-button', !this.isAddButtonVisible() || newMode);
 
 		const header = this.renderHeader();
-		const ITEM_HEIGHT = 24;
-		let listHeight = ITEM_HEIGHT * this.model.items.length;
+		let listHeight = 0;
 
 		if (header) {
-			listHeight += ITEM_HEIGHT;
 			this.listElement.appendChild(header);
+			listHeight += header.clientHeight;
```

Comment:
Left over from rendering the descriptions? I think it's better to avoid using clientHeight